index.html

```diff
@@ -187,20 +83,19 @@ <h1 class="section-heading" id="faq-heading">FAQs</h1>
                     </div>
                     <button class="accordion">Can I win any prizes?</button>
                     <div class="panel">
-                         <p>But of course! We will be giving away up to $850 in prizes this year; you could win up to $50 per person on your team!<!-- You can get a sneak peek of the prizes at <a href="https://quhacks2020.devpost.com/">https://quhacks2020.devpost.com/</a>--></p>
+                         <p>But of course! Prize information will be anounced closer to the event. </p>
                     </div>
                 </div>
                 <div class="faq">
                     <button class="accordion">What do I need to compete?</button>
                     <div class="panel">
                         <p> The only thing you need is a device that can connect to the
-                            internet, and any materials needed for creating your hack! QuHacks
-                            2021 will be held virtually, so you can join from the comfort of your own home.</p>
+                            internet, and any materials needed for creating your hack! </p>
```

Comment:
Let's maybe add something like "a device that can connect to the internet, any materials you need for creating your hack, and yourself!"